lib/store.rb

```diff
@@ -0,0 +1,34 @@
+class Store
+
+  attr_reader :aws_client
+  
+  def initialize
+    app = Application.new
+    @aws_client = app.client
+  end
+  
+  def create_bucket(bucket_name)
+    raise S3StoreArgumentError, "S3StoreArgumentError :: Invalid bucket name.".colorize(:red) unless Bucket.valid?(bucket_name)
```

Comment:
What is the difference between this line and L15?

Reply:
I had kept L15 for false positives in our regex that would go through but would raise error at the s3 end. But that is an extreme edge case and less likely to happen and so I am removing the rescue block on L15.